app/models/Customer.py

```diff
@@ -0,0 +1,12 @@
+class Customer:
+    def __init__(self, custId: int, custName: str, from_location: str, to_location: str):
+        self.custId = custId
+        self.custName= custName
+        self.from_location = from_location
+        self.to_location = to_location
+
```

Comment:
A single customer cannot book multiple rides at the same time, if i am not wrong

Reply:
Open for comments. Can do the suggest changes in the next PR

Reply:
Okay I noticed that you are keeping customers are sort of ephimeral, you are only creating them when booking a ride. But I think later when we will start persisting data, and have separate views for customers, where they can see their ride history and stuff like that, it will make sense to separate out customers from locations. 

I don't see an issue for now.

Reply:
Yeah,  for now it is a composition.